app/services/fees/fee-calculator-errors.go

```diff
@@ -0,0 +1,12 @@
+package fees
```

Comment:
Since this file is in the `fees` package, maybe it would be better to name it simply `errors.go`? I think that it is redundant to have it as `fee-calculator-errors`.

Same thing applies for the `fee-calculator.go` file. It is inside the `fees` package, maybe we could directly name the file `calculator.go`